fake-koji/src/main/java/org/fakekoji/jobmanager/model/TestJob.java

```diff
@@ -87,6 +93,8 @@ public TestJob(
                 platform,
                 variants,
                 buildPlatform,
+                null,
+                null,
```

Comment:
the pull cann not e working afte rthat...

Reply:
why not?

Reply:
nothing in that class can work with null, or can it?